pom.xml

```diff
@@ -75,18 +75,8 @@
         <version.maven-project-info-reports-plugin>2.9</version.maven-project-info-reports-plugin>
         <version.maven-failsafe-plugin>2.22.0</version.maven-failsafe-plugin>
 
-        <md-doclet-group>com.github.iotaledger</md-doclet-group>
```

Comment:
Please keep this group and artifact, as its very inconvenient to switch from local development and testing to Jitpack pom information without it, due to it being needed on multiple places.

Reply:
I added that part again.